framework/doc/content/source/ics/ConstantIC.md

```diff
@@ -1,33 +1,14 @@
 # ConstantIC
 
-!alert! construction title=Undocumented Class
-The ConstantIC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
-```markdown
-# ConstantIC
-
 !syntax description /ICs/ConstantIC
 
-## Overview
+This constant value may be set for the whole variable domain, or restricted to particular blocks or boundaries.
```

Comment:
```suggestion
Sets a constant initial condition described by parameter [!param](/ICs/ConstantIC/value). It can be restricted to particular blocks and boundaries using the [!param](/ICs/ConstantIC/block) and !param](/ICs/ConstantIC/boundary) parameters, respectively.
```